src/Homepage/__snapshots__/Homepage.test.js.snap

```diff
@@ -0,0 +1,63 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`The Homepage component renders correctly 1`] = `
+<div>
+  <img
+    alt=""
```

Comment:
Maybe you forgot it or was it intention: there is no alt text

Reply:
This is intentional. The image is just there for decorative purposes. It doesn't need to be read by a screenreader.